### DISCUSSION TITLE
Unresponsive component on Github site

### DISCUSSION THREAD

**Discussion by emretinaz** [community/community#52257] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

I found an unresponsive component while I browsed on my phone.

Device: iPhone 11 Pro
Browser: Chrome

Page URL: https://github.com/features/copilot#pricing

Section: CTA buttons in Copilot for the Business Card component.

![IMG_8201](https://user-images.githubusercontent.com/20618484/230615409-a71690ec-c86c-4b9d-b8e4-1288321b9747.PNG)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @emretinaz ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
